### DISCUSSION TITLE
Clicking on a search result filepath should open in github.com directly

### DISCUSSION THREAD

**Discussion by neelrr1** [community/community#38055] [Other Feature Feedback, Questions, & Ideas] [answered]
**Note: This discussion is about the `cs.github.com` preview.**

Currently, issuing a search query returns a bunch of matching file snippets with the filepath at the top of the code block.

Intuitively, I expect that clicking on this filepath would open the file in `github.com` not in Github Code Search. My reasoning is that at this point, I've already found what I was looking for and want to continue exploring that code in the context of local files/repo that can be found with the `github.com` UI.

I know that it is possible to first click on the file and then click on the "view on github.com" to open in `github.com` but that extra click is really annoying sometimes.

If there is already a way to do this (go to `github.com` from search results page), please let me know! It would be super helpful 🙂

Overall, I'm really loving the preview.

**Comment by david-engelmann** [comment]
When I search through all of github, navigate to the code section and click on a result, I land on the file within the github.com url for that repository. No extra click needed!

**Reply by neelrr1** [reply]
Sorry, I was trying to give feedback on the beta/technology preview of `cs.github.com`. If there's a better platform for that, I'll repost this there.

**Comment by look** [comment, marked as answer]
@neelrr1 clicking on a result in cs.github.com takes you to the cs.github.com blob view because it includes some new functionality that we wanted to explore, like symbol navigation and a file tree. 

That said, we are working on bringing the new code search and navigation experience to github.com and when that is available, you will of course stay within the github.com site. Stay tuned!

**Reply by look** [reply]
@neelrr1 We have integrated the new code search into GitHub.com, so you will no longer need to go to a separate site to search and the results will, of course, open on GitHub.com. All users who were in the cs.github.com Technology Preview have been added to the new experience, so you should be able to use it.

More details: https://github.com/features/code-search
